static/docs/commands-reference/diff.md

```diff
@@ -39,12 +39,44 @@ SCM, for example when `dvc init` was used with the `--no-scm` option
 
 * `-v`, `--verbose` - displays detailed tracing information.
 
-## Example
+## Examples: Previous version of the same branch
 
 For the setup of our examples we can use the steps in our [Get
-Started](/doc/get-started) guide. Lets assume you've followed them up to the
-[Add Files](https://dvc.org/doc/get-started/add-files) section (or checked out
-the corresponding commit in your Git repo).
+Started](/doc/get-started) guide up to the [Add
+Files](/doc/get-started/add-files) section.
+
+<details>
+
+### Click and expand to setup example
+
+Start by cloning our sample repo if you don't already have it. Then move into
+the repo and checkout the
+[version](https://github.com/iterative/example-get-started/commit/ed10968bcb0dab72563d05712f24ddfff698c87b)
```

Comment:
we have a lot of tags, don't use specific commit hash sums - that repo is auto-generated and commits are subject to change

Reply:
Got it. Using https://github.com/iterative/example-get-started/releases/tag/3-add-file